modules/nf-core/fq/generate/main.nf

```diff
@@ -0,0 +1,33 @@
+process FQ_GENERATE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::fq=0.9.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fq:0.9.1--h9ee0642_0':
+        'quay.io/biocontainers/fq:0.9.1--h9ee0642_0' }"
+
+    input:
+    val meta
+
+    output:
+    tuple val(meta), path("*.R1.fastq.gz"), path("*.R2.fastq.gz"), emit: fastq
+    path "versions.yml"                                          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    fq generate \\
+        $args \\
+        ${prefix}.R1.fastq.gz ${prefix}.R2.fastq.gz
```

Comment:
I don't know if we have official guidelines on this. My personal preference is to us underscores within filenames and dots only to separate file extensions.

Reply:
Yes, I tend to agree with this too. We use underscores in most places so be good to be consistent.